.env.example

```
@@ -16,6 +18,7 @@ MONGO_URL=mongodb://mongo:27017/urbackend
 REDIS_URL=redis://redis:6379
 
 # ── Authentication ────────────────────────────────────────────────────────────
+# Generate random strings for these values (e.g., `openssl rand -base64 32`)
 JWT_SECRET=your_super_secret_jwt_key_min_32_chars
 ENCRYPTION_KEY=32_character_long_string_for_byod_creds
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ENCRYPTION_KEY key should go before the JWT_SECRET key

(UnorderedKey)